src/com/facebook/buck/intellij/ideabuck/META-INF/plugin.xml

```diff
@@ -188,6 +188,15 @@
       <add-to-group anchor="last" group-id="MainToolBar"/>
     </group>
 
+    <group id="buck.test">
+      <action id="buck.RunSelectedTest"
```

Comment:
Lets create tow actions

"Run Test(s) with Buck"
"Debug Test(s) with Buck"

and add the buck icon to the left side